Models/Yukawa/yukawa.py

```diff
@@ -228,180 +152,248 @@ def evaluate(
-        that the effective potential and particle mass functions refer to
-        model.modelParameters, so be careful not to replace that reference here.
-        """
+    solverSettings = WallGo.WallSolverSettings(
+        bIncludeOffEquilibrium=False,
+        # meanFreePathScale is determined here by the annihilation channels,
+        # and scales inversely with y^4 or lam^2. This is why
+        # meanFreePathScale has to be so large.
+        meanFreePathScale=10000.0,  # In units of 1/Tnucl
+        wallThicknessGuess=10.0,  # In units of 1/Tnucl
+    )
 
-        # oldParams = model.modelParameters.copy()
-        model.updateModel(inputParameters)
+    results = manager.solveWall(solverSettings)
 
-    def getBenchmarkPoints(self) -> list[ExampleInputPoint]:
-        """
-        Input parameters, phase info, and settings for the effective potential and
-        wall solver for the Yukawa benchmark point.
-        """
+    print(
+        f"Wall velocity without out-of-equilibrium contributions {results.wallVelocity:.6f}"
+    )
 
-        output: list[ExampleInputPoint] = []
-        output.append(
-            ExampleInputPoint(
-                {
-                    "sigma": 0.0,
-                    "msq": 1.0,
-                    "gamma": -1.2,
-                    "lam": 0.10,
-                    "y": 0.55,
-                    "mf": 0.30,
-                },
-                WallGo.PhaseInfo(
-                    temperature=8.0,  # nucleation temperature
-                    phaseLocation1=WallGo.Fields([0.4]),
-                    phaseLocation2=WallGo.Fields([27.0]),
-                ),
-                WallGo.VeffDerivativeSettings(
-                    temperatureVariationScale=1.0,
-                    fieldValueVariationScale=[
-                        100.0,
-                    ],
-                ),
-                WallGo.WallSolverSettings(
-                    # we actually do both cases in the common example
-                    bIncludeOffEquilibrium=True,
-                    # meanFreePathScale is determined here by the annihilation channels,
-                    # and scales inversely with y^4 or lam^2. This is why
-                    # meanFreePathScale has to be so large.
-                    meanFreePathScale=10000.0, # In units of 1/Tnucl
-                    wallThicknessGuess=10.0, # In units of 1/Tnucl
-                ),
-            )
-        )
+    solverSettings.bIncludeOffEquilibrium = True
 
-        return output
+    results = manager.solveWall(solverSettings)
 
-    # ~ End WallGoExampleBase interface
+    print(
+        f"Wall velocity with out-of-equilibrium contributions {results.wallVelocity:.6f}"
+    )
 
 
+## Don't run the main function if imported to another file
 if __name__ == "__main__":
-
-    example = YukawaModelExample()
-    example.runExample()
+    main()
+
+
+# class YukawaModelExample(WallGoExampleBase):
```

Comment:
This should obviously be deleted, not just commented out. I will update it on my branch